src/azure-cli/HISTORY.rst

```diff
@@ -7,6 +7,10 @@ Release History
 
 * Add support for importing/exporting feature flags
 
+**SQL**
+
+* New commands `sql db sensitivity-labels show/list/list-recommended/update/delete/enable-recommendation/disable-recommendation` to manage sensitivity labels for SQL databases.
```

Comment:
Can you consider making another command group `az sql db sensitivity-labels recommendation`, which has `list`, `enable` and `disable` commands inside to replace your `list-recommended`, `enable-recommendation` and `disable-recommendation` commands?